src/main/java/org/folio/marccat/resources/domain/AuthorityRecord.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.marccat.resources.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import javax.annotation.Generated;
+
+/**
+ * @author elena
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Generated("org.jsonschema2pojo")
+@JsonPropertyOrder({"id", "group", "canadianContentIndicator", "verificationLevel", "leader", "fields", "recordView"})
+public class AuthorityRecord extends Record{
```

Comment:
```suggestion
public class AuthorityRecord extends Record {
```
There are other places where there is no space before the `{`. We should correct them all.

Reply:
I have formatted the files